npm run lint 시 에러가 발생하여 조금 수정했습니다.
`C:\Users\Moonding\kaist_fed\billboard.js\src\ChartInternal\shape\polar.ts

  **23:1   error  This line has a length of 103. Maximum allowed is 100  max-len**
  23:43  error  Operator '||' must be spaced                           space-infix-ops
  24:1   error  Trailing spaces not allowed                            no-trailing-spaces`

--- a/demo/demo.js
+++ b/demo/demo.js
@@ -584,6 +584,9 @@ var demos = {
 					level:{
 						depth: 4,
 					},
+					size:{
+						max: 200,
+					}
 				}
 			}
 		},

--- a/src/ChartInternal/shape/polar.ts
+++ b/src/ChartInternal/shape/polar.ts
@@ -20,7 +20,8 @@ export default {
 			.attr("class", CLASS.chartPolarArcs);
 
 		// Let every data is less or equal to dataMax and each level has integer value
-		current.dataMax = Math.ceil($$.getMinMaxData().max[0].value / depth) * depth;
+		current.dataMax = config.polar_size_max ||Math.ceil($$.getMinMaxData().max[0].value / depth) * depth;
+		
 
 		// Let each value be 1, thus every arc has same central angle
 		// To match central angle with specific data, change "1" to specific function.

--- a/src/ChartInternal/shape/polar.ts
+++ b/src/ChartInternal/shape/polar.ts
@@ -7,6 +7,7 @@ export default {
 		const $$ = this;
 		const {config, state: {current}, $el} = $$;
 		const depth = config.polar_level_depth;
+		const ceilDataMax = Math.ceil($$.getMinMaxData().max[0].value / depth) * depth;
 
 		$el.polar = $el.main.select(`.${CLASS.chart}`).append("g")
 			.attr("class", CLASS.chartPolars);
@@ -20,9 +21,7 @@ export default {
 			.attr("class", CLASS.chartPolarArcs);
 
 		// Let every data is less or equal to dataMax and each level has integer value
-		current.dataMax = config.polar_size_max ||Math.ceil($$.getMinMaxData().max[0].value / depth) * depth;
-		
-
+		current.dataMax = config.polar_size_max || ceilDataMax;
 		// Let each value be 1, thus every arc has same central angle
 		// To match central angle with specific data, change "1" to specific function.
 		$$.polarPie = d3Pie().value(1);

--- a/src/config/Options/shape/polar.ts
+++ b/src/config/Options/shape/polar.ts
@@ -15,13 +15,18 @@ export default {
 	 * @type {object}
 	 * @property {object} polar Polar object
 	 * @property {number} [polar.level.depth=3] Set the level depth.
+	 * @property {number} [polar.size.max=undefined] Set the chart size. Same as setting the max value of the polar chart. If not given, it'll take the max value from the given data.
 	 * @see [Demo](https://naver.github.io/billboard.js/demo/#Chart.PolarChart)
 	 * @example
 	 *  polar: {
 	 * 		level: {
 	 * 			depth: 3
+	 * 		},
+	 * 		size: {
+	 * 			max: 200
 	 * 		}
 	 *  }
 	 */
 	polar_level_depth: 3,
+	polar_size_max: <number|undefined> undefined,
 };

